### ISSUE TITLE
Gestion des préférences des utilisateurs

### ISSUE THREAD

**Issue by ChantyTaguan** [ChantyTaguan/zds-site#10]
Pour chaque type d'abonnement existant, l'utilisateur doit pouvoir paramétrer si par défaut il veut recevoir un email en plus de la notification sur le site.

**Comment by ChantyTaguan** [comment]
Par défaut : uniquement notification sur le site ? Ou comportement par défaut différent selon la notification (par exemple, par défaut, email pour la notification d'une mise à jour d'n tuto/article par un co-auteur)

**Comment by ChantyTaguan** [comment]
Est-ce qu'on donne aussi la possibilité de recevoir uniquement un email, et rien sur le site ?

**Comment by Situphen** [comment]
> Par défaut : uniquement notification sur le site ? Ou comportement par défaut différent selon la notification (par exemple, par défaut, email pour la notification d'une mise à jour d'n tuto/article par un co-auteur)

Je mettrai par défaut notification sur le site, rien par courriel !

> Est-ce qu'on donne aussi la possibilité de recevoir uniquement un email, et rien sur le site ?

Je pense que la majorité des personnes qui demandent à recevoir un courriel, c'est pour être au courant et donc elles n'ont pas envie de désactiver les notifications sur le site. 'Fin, ce serait illogique... Donc je ne peine pas que ce soit la peine de développer ça.

**Comment by ChantyTaguan** [comment]
Oui, ça me semble aussi relativement inutile de chipoter. Si y a 8000 membres qui viennent dire qu'ils veulent pouvoir être notifiés uniquement par email on pourra toujours développer ça par après mais j'en doute...

**Comment by GerardPaligot** [comment]
1. Vu toutes les choses à laquelle l'utilisateur pourra s'abonner, la notification par e-mail me semble une mauvaise idée. Il y a des risques de spam dans la boite mail des membres **et** donc surtout, une identification des boites mail des mails de Zeste de Savoir comme des spams. Chose que nous devrions éviter.
2. Les notifications par site doivent être toujours activées mais lorsqu'on accède à un contenu souscrit par e-mail (ou non d'ailleurs), il faut que la notification du site disparait.

**Comment by ChantyTaguan** [comment]
D'accord avec 1 et 2 :)

**Comment by Situphen** [comment]
D'ailleurs, il faudrait un lien en bas de chaque courriel pour se désinscrire en un clic d'un abonnement.

**Comment by GerardPaligot** [comment]
C'est pas bête !
